modules/images/webp-uploads/load.php

```diff
@@ -741,3 +741,43 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Register media settings field to generate multiple mime WebP & JPEG output.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_register_media_settings_field() {
+	// Register field setting for the processing.
+	register_setting( 'media', 'generate_webp_and_jpeg' );
```

Comment:
See above, please prefix with `perflab_` throughout this PR.

More importantly though, please use the full setting registration, also providing the necessary `$args`. Settings should be registered on `init`, not `admin_init` (the `add_settings_field()` call below should remain in `admin_init` though).

Reply:
Done.